pubspec.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
   js: ^0.6.1+1
   matcher: ^0.12.1+4
   meta: ^1.1.0
-  over_react: ^3.5.3
+  over_react: ">=3.5.3 <5.0.0"
   react: ^5.5.1
```

Comment:
This also needs to be widened

Reply:
Ahh yep, thank you @greglittlefield-wf! Updated!